src/mlio/default_instance_reader.h

```diff
@@ -39,6 +39,8 @@ class default_instance_reader final : public instance_reader_base {
                                      record_reader_factory &&fct);
 
 private:
+    bool corrupt_split_record_;
```

Comment:
Please move this to the bottom of the class definition where by convention all private variables are listed together.